src/frontend/packages/lib_video/src/components/common/VideoWidgetProvider/widgets/VODCreation/index.tsx

```diff
@@ -56,28 +56,20 @@ export const VODCreation = () => {
   if (video.live_state === liveState.HARVESTING) {
     content = (
       <Box fill background="white">
-        <Box direction="row" margin="auto">
-          <Paragraph
-            margin={{ vertical: 'auto', right: 'small' }}
-            color="blue-active"
-            textAlign="center"
-          >
+        <Box direction="row" margin="auto" align="center">
+          <Text color="blue-active" className="mr-s" textAlign="center">
             {intl.formatMessage(messages.harvestingInProgress)}
-          </Paragraph>
+          </Text>
           <Spinner margin={{ horizontal: 'auto' }} />
         </Box>
       </Box>
     );
   } else if (displayMessageLayer) {
     content = (
       <Box fill background="white">
-        <Paragraph
-          margin={{ horizontal: 'auto', vertical: 'small' }}
-          color="blue-active"
-          textAlign="center"
-        >
+        <Text color="blue-active" textAlign="center">
```

Comment:
Type p ?

Reply:
Same, it is an inline text, should be a span.